flake/dev/generate-all-maintainers.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, ... }:
+{
+  perSystem =
+    { pkgs, config, ... }:
+    {
+      packages.generate-all-maintainers =
+        let
+          meta = pkgs.callPackage ../../stylix/meta.nix { };
+          all_maintainers = lib.pipe meta [
+            lib.attrsToList
+            (map ({ value, ... }: value.maintainers or [ ]))
+            builtins.concatLists
+            builtins.toJSON
+          ];
+        in
+        pkgs.runCommand "generate-all-maintainers" { inherit all_maintainers; } ''
```

Comment:
This will likely run into bash argument length errors.

Best to pass the JSON in as a file, rather than a string